easybuild/tools/filetools.py

```diff
@@ -408,7 +508,7 @@ def calc_block_checksum(path, algorithm):
         blocksize = algorithm.blocksize * 262144  # 2^18
     except AttributeError, err:
         blocksize = 16777216  # 2^24
-    _log.debug("Using blocksize %s for calculating the checksum" % blocksize)
+    _log.debug("Using blocksize %s for calculating the checksum", blocksize)
```

Comment:
ok, I now understand it doesn't work yet with `log.error`, which may explain what I've been seeing with running into uncompleted template strings; we should fix our custom `log.error` then...